src/kOS/Sound/Voice.cs

```diff
@@ -0,0 +1,197 @@
+﻿using System;
+using UnityEngine;
+using kOS.Safe.Sound;
+
+
+namespace kOS.Sound
+{
+	/// <summary>
```

Comment:
I swear I told it to always use spaces instead of tabs.  I recently upgraded my copy of SharpDevelop.  Maybe in so doing I lost that setting.